### PR TITLE
Trie sync constant extraction

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- adjusting the constant used in the trie syncer was a cumbersome process that required binary recompilation. Also, a new, better value was found during mainnet testing.
  
## Proposed Changes
- extracted constant, adjusted it to ~10ms. Original value was ~100ms.

## Testing procedure
- standard testing procedure
- mainnet sync comparison

### CODE REVIEW

File: trie/sync.go [hunk 1]
```diff
@@ -80,7 +83,7 @@ func NewTrieSyncer(arg ArgTrieSyncer) (*trieSyncer, error) {
 		nodesForTrie:              make(map[string]trieNodeInfo),
 		topic:                     arg.Topic,
 		shardId:                   arg.ShardId,
-		waitTimeBetweenRequests:   time.Second,
+		timeBetweenRechecks:       arg.TimeBetweenRechecks,
```
Comment (bogdan-rosianu) [inline trie/sync.go:86]: is this intended? moving from 1s to 10ms is a pretty big difference

**Review by bogdan-rosianu — APPROVED** [review]

File: trie/sync.go [hunk 2]
```diff
@@ -119,6 +122,15 @@ func checkArguments(arg ArgTrieSyncer) error {
 		return fmt.Errorf("%w provided: %v", ErrInvalidMaxHardCapForMissingNodes, arg.MaxHardCapForMissingNodes)
 	}
 
+	return CheckTimeBetweenRechecks(arg.TimeBetweenRechecks)
+}
+
+// CheckTimeBetweenRechecks will check the provided value against the minimum allowed
+func CheckTimeBetweenRechecks(timeBetweenRechecks time.Duration) error {
+	if timeBetweenRechecks < minTimeBetweenRechecks {
+		return fmt.Errorf("%w for TimeBetweenRechecks", ErrInvalidValue)
```
Comment (BeniaminDrasovean) [inline trie/sync.go:131]: Could also print `timeBetweenRechecks` and `minTimeBetweenRechecks`

**Review by BeniaminDrasovean — DISMISSED** [review]

File: trie/sync.go (hunk 1 above)
Comment (iulianpascalau) [reply trie/sync.go:86]: yes, v1 of the trie sync is not used for more than 1 1/2 years. Should have been deleted anyway

Comment (iulianpascalau) [reply trie/sync.go:86]: To answer your question, this adjustment should make the v1 trie sync variant better

**Review by bogdan-rosianu — APPROVED** [review]

**Review by BeniaminDrasovean — APPROVED** [review]

**Pull request closed without merging**